sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -2044,6 +2044,13 @@ object RapidsConf {
         "The gpu to disk spill bounce buffer must have a positive size")
       .createWithDefault(128L * 1024 * 1024)
 
+  val NON_UTC_TIME_ZONE_ENABLED = 
```

Comment:
FYI. No need to introduce a configuration here. We can use the approach from https://github.com/NVIDIA/spark-rapids/pull/9719.

Reply:
Per internal discussion, for 23.12 release, we are putting current timezone work behind a configuration flag to avoid risk for release.